experiments/compositions/composition/internal/controller/expander_reconciler.go

```diff
@@ -329,65 +332,24 @@ func (r *ExpanderReconciler) SetupWithManager(mgr ctrl.Manager, cr *unstructured
 		Complete(r)
 }
 
-func (r *ExpanderReconciler) getExpanderVersion(ctx context.Context, inputExpanderVersion string, expanderType string) (string, string, error) {
+func (r *ExpanderReconciler) getExpanderVersion(ctx context.Context, inputExpanderVersion string, expanderType string) (string, *compositionv1alpha1.ExpanderVersion, error) {
```

Comment:
If the version is not specified and defaults to latest, how does this function gets the latest image?

Reply:
The `status.versionMap["latest"]` is populated by the reconciler.